### ISSUE TITLE
React front-end accessibility issues [dev_react]

### ISSUE THREAD

**Issue by AdamLeonSmith** [Path-Check/safeplaces-ct-frontend#29]
- [ ] The aria-label is missing on the time slider (.rc-slider-handle-1 and .rc-slider-handle-2)
- [ ] Form element [#react-select-2-input] does not have an aria-label
- [ ]  Actions drop-down; "Select..." default text; all the blue buttons -  do not have sufficient colour contrast to meet WCAG guidelines.  See https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=AxeChrome

**Comment by Mesum387** [comment]
I can do it, assign me
